src/app/olx-date-format.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
```

Comment:
Remove this class. You can pass parameters to the DatePipe directly. See [DatePipe docs](https://angular.io/docs/ts/latest/api/common/index/DatePipe-pipe.html)

Also why call it Olx? If this pipe would have been useful and you want to avoid naming conflicts, consider giving it a meaningful name, i.e. "FinnishDatePipe".